<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RVR_CurrentStatus</class>
 <widget class="QDialog" name="RVR_CurrentStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>821</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>当前状态</string>
   </property>
   <widget class="QGroupBox" name="groupBox_recvi">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>接收机</string>
    </property>
    <widget class="QLabel" name="label_reciv_temp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>26</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reciv_hum">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>26</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_reciver_temperature">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_reciver_humidity">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>431</width>
      <height>501</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>431</width>
      <height>501</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>图像</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="WindMeter" name="Wind_widget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_send">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>发射机</string>
    </property>
    <widget class="QLabel" name="label_send_temp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>26</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_send_hum">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>26</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_send_temperature">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_send_humidity">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>大气透射仪</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>24</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>RVR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_RVR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>98</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>23</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>MOR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Mor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_AD">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>870</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>870</y>
       <width>18</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>AD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>18</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>18</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_WIN">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>8</y>
      <width>39</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>WIND</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HT">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>8</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HumTemp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RV">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>8</y>
      <width>27</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RVR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Rstate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>12</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_windState">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>12</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_htState">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>12</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WindMeter</class>
   <extends>QWidget</extends>
   <header location="global">windPlugins/windmeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
